test/runtests.jl

```diff
@@ -50,9 +50,10 @@ end
 gpu_broken || @safetestset "experiment test: CoupledSims tests" begin
     include("experiment_tests/coupled_sims.jl")
 end
-@safetestset "experiment test: Leaderboard" begin
-    include("experiment_tests/leaderboard.jl")
-end
+# This test fails because of the undownloadable artifact
+# @safetestset "experiment test: Leaderboard" begin
+#     include("experiment_tests/leaderboard.jl")
+# end
```

Comment:
@szy21 @Sbozzolo I just saw that this is commented out - do we have plans to fix it?

Reply:
There's an issue about that. It'll be fixed automatically once Kevin merges his changes